submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                    With an <span class="to-orange">AVERAGE</span> timeline set
+                    up, our expert translator can take a reasonable amount of
+                    time perfecting your translation
+                  </p>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="payment-method">
+            <div class="allTextDocumentUpload">
+              <span class="numberSection"><b>5</b></span>
+              <h2 class="uploadText">Select your payment method</h2>
+            </div>
+            <div class="block-to-center">
+              <div>
+                <h5 class="smallUploadText">
+                  Already have an account with us?
+                  <a href="#" class="underlined-a">Sign in.</a>
+                </h5>
+                <h5 class="above-text">FULL NAME</h5>
+                <input
+                  type="text"
+                  id="text-name"
+                  class="text text-name"
+                  name="Full Name"
+                  placeholder="Mattew Alebard"
+                  title="Enter your name. Minimal length 3 characters."
+                  minlength="3"
+                  required
+                /><span class="validation-sign"></span>
+              </div>
+            </div>
+            <div class="block-to-center">
+              <div class="field-text">
+                <div class="field-container">
+                  <h5 class="above-text">EMAIL</h5>
+                  <input
+                    type="email"
+                    id="email"
+                    class="text text-name"
+                    name="email"
+                    placeholder="example@domain.com"
+                    pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+                    title="Enter your email. Example: name@domain.com"
+                    required
+                  /><span class="validation-sign"></span>
+                </div>
+                <div class="field-container">
+                  <h5 class="above-text">CHOOSE PASSWORD</h5>
+                  <input
+                    type="password"
+                    id="password"
+                    class="text text-name"
+                    name="password"
+                    placeholder="Enter your password"
+                    pattern="(?=^.{8,}$)((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*"
+                    title="Enter your password. Minimum 8 characters, UPPERCASE, lowercase and Number."
+                    minlength="6"
+                    required
+                  /><span class="validation-sign"></span>
+                </div>
+              </div>
+              <div class="payment-method-choose">
+                <div class="payholder">
+                  <label for="card">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="card"
```

Comment:
Id - it's a unique name. It must be only one on all page. In your case, you have three elements with the same id name.